homework3/task1.js

```diff
@@ -0,0 +1,22 @@
+function pickRandomNum(randomArr, numPick) {
+  let result = [];
```

Comment:
don't need `let` here
please use `let` only in that case when you need to rewrite your variable